pkg/api/defaults_test.go

```diff
@@ -1077,45 +1077,94 @@ func TestAgentPoolProfile(t *testing.T) {
 func TestDistroDefaults(t *testing.T) {
 
 	var tests = []struct {
-		name                string              // test case name
-		orchestratorProfile OrchestratorProfile // orchestrator to be tested
-		expectedDistro      Distro              // expected result default disto to be used
+		name                   string              // test case name
+		orchestratorProfile    OrchestratorProfile // orchestrator to be tested
+		masterProfileDistro    Distro
+		agentPoolProfileDistro Distro
+		expectedDistro         Distro // expected result default disto to be used
```

Comment:
Are we testing both agent profile and master profile distro results?

Reply:
we are but the test was setup to have a common distro for both

Reply:
I can change it to have distinct expected values